content/english/debugging/_index.md

```diff
@@ -14,12 +14,12 @@ So, you’ve learned the basics of a programming language. You’ve probably mas
 
 However, what happens when your code fails to compile? Beginners to programming usually search online to see if others have encountered similar problems. This is a bit tedious, as you'd need to turn to the internet every single time you run into an error. 
 
-What happens when your program compiles, but doesn't work the way you expect it to (i.e., you found **a bug? 🪲)**
+What happens when your program compiles, but doesn't work the way you expect it to? (i.e., you found **a bug? 🪲)**
 
 Going back to fix your code can be a challenge more difficult than writing the code itself. Learning how to navigate these bugs and errors takes skill, patience, and experience. In this workshop, we'll unravel the basic process for debugging your code. Remember that debugging is part of all programming languages.
 
 {{% panel theme="info" header="Why call it a bug?"%}}
-The first recorded programming bug was recorded all the way back in 1947 when a moth decided to be in the wrong place. Check out [The World’s First Computer Bug.](https://education.nationalgeographic.org/resource/worlds-first-computer-bug)
+The first recorded programming bug was recorded all the way back in 1947 when a moth decided to be in the wrong place. Check out <a href="https://education.nationalgeographic.org/resource/worlds-first-computer-bug" target="_blank">The World’s First Computer Bug.</a>
 
 {{% /panel %}}

```

Comment:
Can we put the replit link on line 28 into a <a target="_blank"> element?

Reply:
There is already a "Launch Replit" button below, it may be confusing to have 2 elements when the existing one should already prompt for a login. Can change in future PR when switching from replit to dotnetfiddle